README.md

```diff
@@ -36,12 +36,17 @@ build one with its internal builder:
 ```java
 Request request = Request.builder()
         .method("POST")
-        .uriWithQuery(URI.create("https://localhost/service/v2/users"))
-        .body("{ field: \"foo\" }")
-        .header("Content-Type", "application/json")
+        .uriWithQuery(URI.create("https://akaa-baseurl-xxxxxxxxxxx-xxxxxxxxxxxxx.luna.akamaiapis.net/billing-usage/v1/reportSources"))
```

Comment:
That should be a relative URI, because host part will be taken from `ClientCredential`. Otherwise, it is confusing. A user may ask: Why do I have same host both in request and in credential?

Perhaps we should even call it `relativeUriWithQuery` and validate when a user constructs it.